assets/opstools/AppBuilder/classes/platform/views/ABViewCSVImporter.js

```diff
@@ -1209,6 +1209,19 @@ module.exports = class ABViewCSVImporter extends ABViewCSVImporterCore {
 
             /** Prepare Data */
             let parsedData = [];
+            function escapeHtml(text) {
+               var map = {
+                  "&": "&amp;",
+                  "<": "&lt;",
+                  ">": "&gt;",
+                  '"': "&quot;",
+                  "'": "&#039;"
+               };
+
+               return text.replace(/[&<>"']/g, function(m) {
```

Comment:
I like this. I just know the second parameter of .replace function can be a function.